### PR TITLE
chore: delete transformed checks from cron

### PR DESCRIPTION
**Pull request by yashmehrotra**
Fixes: https://mission-control.azure.lab.flanksource.com/incidents/018929f8-be24-2368-7983-a4e47955d5b5

### CODE REVIEW

File: pkg/jobs/canary/canary_jobs.go [hunk 1]
```diff
@@ -51,7 +51,8 @@ func StartScanCanaryConfigs(dataFile string, configFiles []string) {
 type CanaryJob struct {
 	*kommons.Client
 	Kubernetes kubernetes.Interface
-	v1.Canary
+	Canary     v1.Canary
+	DBCanary   pkg.Canary
```
Comment (yashmehrotra) [inline pkg/jobs/canary/canary_jobs.go:55]: Will use DB object from now on, will slowly deprecate uses of the old spec

File: pkg/jobs/canary/canary_jobs.go [hunk 2]
```diff
@@ -111,6 +112,11 @@ func (job CanaryJob) Run() {
 		logger.Errorf("error deleting transformed checks for canary %s: %v", job.Canary.GetPersistedID(), err)
 	}
 
+	if len(checksToMarkHealthy) > 0 {
+		// Remove from cron and let the transformed checks repopulate
```
Comment (moshloop) [inline pkg/jobs/canary/canary_jobs.go:116]: Not sure I understand this- Why do we remove the schedule?

Comment (yashmehrotra) [reply pkg/jobs/canary/canary_jobs.go:116]: When a transformed check is marked healthy, it is technically soft deleted but it still remains in the cronjob. We delete the cronentry and the SyncCanaryJob repopulates the cron with non deleted checks

File: pkg/db/canary.go [hunk 3]
```diff
@@ -96,10 +96,6 @@ func PersistCheck(check pkg.Check, canaryID uuid.UUID) (uuid.UUID, error) {
 		"deleted_at":  nil,
 	}
 
-	// Deletions for transformed checks are handled separately
```
Comment (yashmehrotra) [inline pkg/db/canary.go:99]: @moshloop 
Not sure why we put this, removing it

**Pull request merged**